tests/BuildScriptGenerator.Tests/DefaultBuildScriptGeneratorTest.cs

```diff
@@ -15,6 +16,8 @@ namespace Microsoft.Oryx.BuildScriptGenerator.Tests
 {
     public class DefaultBuildScriptGeneratorTest : IClassFixture<TestTempDirTestFixture>
     {
+        private const string TestPlatformName = "test";
```

Comment:
can you also add a test where checkers can fail but we are not breaking the oryx build for that